topics/request.md

```diff
@@ -1,316 +1,116 @@
-[//]: # (title: Request)
+[//]: # (title: Send a request)
 
-<include src="lib.md" include-id="outdated_warning"/>
+After adding the necessary [dependencies](client.md#add-dependencies) and [creating the client](client.md#create-client), you can send HTTP requests. You can configure the following request parameters:
```

Comment:
It's not entirely clear the phrasing here about the necessary dependencies, because it feels like "request" has its own dependency. Does it make sense for every topic to mention this "After adding...."?

Also, and use of "send" should be "make"

Reply:
I just want to emphasize that a user needs to set up the client before using it. Maybe, it's  better to rephrase to something like: 'After you've set up the client' (with a link to 'Setting up a client' topic).